docs/usage/request-templates.rst

```diff
@@ -0,0 +1,159 @@
+.. _request-templates:
+
+=================
+Request templates
+=================
+
+:ref:`Request templates <request-template-api>` are :ref:`items <items>` that
+allow writing reusable code that creates :class:`~zyte_common_items.Request`
+objects from parameters.
+
+Using request templates
+=======================
+
+After you :ref:`write a request template page object
+<custom-request-template-page>` for a website, you can get a request template
+item for that website and call its ``request`` method to build a request with
+specific parameters.
+
+For example, after implementing the page object :ref:`below
+<custom-request-template-page>`, you can get a
+:class:`~zyte_common_items.SearchRequestTemplate` object for
+https://quotes.toscrape.com/, and build a search request for a specific
```

Comment:
It's unfortunate that our website doesn't allow to have a nice search example. I'd expect the most common case to be a search form which generates a GET request, with added `q={keyword}`, not a tag lookup.

Do you think it'd be possible to have a simpler example here? Maybe even not using a real website.